### PR TITLE
#4765 [Manager]RestartWorkspace errors will not be handled

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4765 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/store/epics/broker/updateBrokerEpic.js [hunk 1]
```diff
@@ -18,34 +18,66 @@ import { normalize } from 'normalizr';
...
+
+// Note: The caller SHOULD handle the error of this action
+export const updateBroker$ = values => {
+  const brokerId = getId(values);
+  return defer(() => brokerApi.update(values)).pipe(
+    map(res => res.data),
+    map(data => normalize(data, schema.broker)),
+    map(normalizedData => merge(normalizedData, { brokerId })),
+    map(normalizedData => actions.updateBroker.success(normalizedData)),
+    startWith(actions.updateBroker.request({ brokerId })),
```
Comment (saivirtue) [inline ohara-manager/client/src/store/epics/broker/updateBrokerEpic.js:63]: We could reuse these part logic and refactor this in #4929 😄

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**